3.10/installation-linux.md

```diff
@@ -1,38 +1,59 @@
 ---
 layout: default
-description: To install ArangoDB on Linux visit the Download page of the ArangoDB and download the correct package for your Linux distribution. Follow the instructions to use your favorite package.
-title: Installing ArangoDB on Linux 
+description: Download an installation or tar package, or use a package manager
+title: Installing ArangoDB on Linux
 ---
 Installing ArangoDB on Linux
 ============================
 
-To install ArangoDB on Linux:
+You can install ArangoDB on most common Linux distributions. The basic
+installation steps are:
 
-1. Visit the official [Download](https://www.arangodb.com/download){:target="_blank"} page of the
-   ArangoDB web site and download the correct package for your Linux distribution.
-   You can find binary packages for the most common distributions there. Linux Mint: 
-   please use the corresponding Ubuntu or Debian packages. 
-2. Follow the installation instructions on the _Download_ page to use your
-   favorite package manager for the major distributions. After setting up the ArangoDB
-   repository you can easily install ArangoDB using _yum_, _aptitude_, _urpmi_ or _zypper_.
+1. Visit the official [Download](https://www.arangodb.com/download){:target="_blank"}
+   page of the ArangoDB web site.
 
-{% hint 'info' %}
-In addition to installation packages (distribution dependent) a `tar.gz` archive
-is available starting from version 3.4.0.
-{% endhint %}
+2. Click the logo of the distribution that matches your operating system.
+   If you use Linux Mint, click Ubuntu or Debian.
+
+3. You can choose between different installation methods and packages:
+   - distribution-dependent installation packages (`.rpm`, `.deb`)
+   - tar packages (`tar.gz` archives)
+   - installation via a package manager
+
+4. Installation and tar packages: You may verify the integrity of a download
+   by comparing the SHA256 hash listed on the website with the hash of the file.
+   For example, you can you run `openssl sha256 <filename>` or
+   `sha256sum <filename>` in a terminal.
+
+   Package manager: package managers generally validate downloaded packages
+   automatically. For more information, see
+   [SecureApt](https://wiki.debian.org/SecureApt){:target="_blank"} (Debian packages) and
+   [Secure distribution of RPM packages](https://www.redhat.com/en/blog/secure-distribution-rpm-packages){:target="_blank"}
+   for instance.
+
+5. Installation packages: run `sudo rpm -i <filename>.rpm` or
+   `sudo apt install <filename>.deb` respectively in a terminal and follow the
+   on-screen instructions.
```

Comment:
`sudo dpkg -i <filename>.deb` could be better.

Reply:
no, dpkg is a development tool. apt is the way to go.

Reply:
apt is generally better because it installs dependencies too, whereas with dpkg, you would need to run `apt install -f` after `dpkg -i ...` to do that - not in our case (static binaries), however.